launcher_scripts/nemo_launcher/core/v2/config_k8s.py

```diff
@@ -1,24 +1,22 @@
-from pydantic import BaseModel, validator, Field, computed_field, model_validator
+from enum import Enum
+from typing import Optional, Union, overload
+
+from hera.workflows.models import (  # TODO: Use these BaseModels when solution we have solution to: https://github.com/argoproj-labs/hera/issues/984; NFSVolumeSource,; HostPathVolumeSource,; PersistentVolumeClaimVolumeSource,
+    Volume,
+    VolumeMount,
+)
+from hydra.utils import get_class
 from kubernetes.client import (
-    V1VolumeMount,
```

Comment:
In general the re-org of the imports is fine with me, but what triggered this large re-org? Is it because you used `isort` separately?

Reply:
The import re-org wasn't really intentional, I just ran formatting via pre-commit and it caused several changes. Main purpose was just to change '|' to 'Union' or 'Optional'

Reply:
`isort` and black are actually creating issues in pre-commit now because they have different line length defaults. Should we just remove `isort` from pre-commit?

Reply:
It looks like isort can be configured to black's defaults:
https://pycqa.github.io/isort/docs/configuration/profiles.html

Example:
```
-   repo: https://github.com/pycqa/isort
    rev: 5.5.4
    hooks:
      - id: isort
        args: ["--profile", "black"]
```
Does that solve the issue you faced?

Reply:
it does not strangely. But setting the line length to be the same as Black (88) does fix.